site_scons/site_tools/doneapi.py

```diff
@@ -0,0 +1,83 @@
+"""SCons.Tool.doneapi
+
+Hack to support oneapi version of Intel compilers
+
+"""
+import sys, os
+
+is_linux = sys.platform.startswith('linux')
+
+if is_linux:
+    import SCons.Tool.gcc
+import SCons.Util
```

Comment:
(pylint-no-name-in-module) No name 'Util' in module 'SCons'
(pylint-import-error) Unable to import 'SCons.Util'